client/dist/styles.css

```diff
@@ -594,13 +598,14 @@ Overview Structure */
 }
 .more-questions:hover {
   color: white;
-  background-color: crimson;
+  background-color: red;
```

Comment:
I think that crimson looks a lot better, Kind of wish we went for a color palette that could use it. But I do think it's better to stay consistent.